operator/controllers/keda_controller_test.go

```diff
@@ -0,0 +1,100 @@
+package controllers
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	rtypes "github.com/kyma-project/module-manager/operator/pkg/types"
+
+	"github.com/kyma-project/keda-manager/operator/api/v1alpha1"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("Keda controller", func() {
+	Context("When creating fresh instance", func() {
+		const (
+			namespaceName = "keda"
+			kedaName      = "test"
+			saName        = "keda-manager"
+		)
+
+		var (
+			emptyState = rtypes.State("")
+		)
+
+		It("The status should be Success", func() {
+
+			ctx := context.Background()
+
+			By(fmt.Sprintf("Creating namespace: %s", namespaceName))
+			namespace := corev1.Namespace{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: namespaceName,
+				},
+			}
+			Expect(k8sClient.Create(ctx, &namespace)).Should(Succeed())
+
+			By(fmt.Sprintf("Creating crd: %s", kedaName))
+			keda := v1alpha1.Keda{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      kedaName,
+					Namespace: namespaceName,
+				},
+				Spec: v1alpha1.KedaSpec{},
+			}
+
+			Expect(k8sClient.Create(ctx, &keda)).Should(Succeed())
+
+			var sa corev1.ServiceAccount
```

Comment:
Now I see what is `sa`, but I think that short names like this are bad idea. We save few letters and waste reader time.